File: python/raindrops/raindrops.py
SOUNDS = {
  3: "Pling",
  5: "Plang",
  7: "Plong",
}

def convert(number):
  

  rain = ""

  for (key, value) in SOUNDS.items():
    if number % key == 0:
      rain += value

  return rain or str(number)

File: javascript/raindrops/raindrops.js
//
// This is only a SKELETON file for the 'Raindrops' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

import { strict } from "assert";

const RAIN = {
  3: "Pling",
  5: "Plang",
  7: "Plong",
};

export const convert = (n) => {
  let rain = "";

  for (const [key, value] of Object.entries(RAIN)) {
    if (n % key === 0) {
      rain += value;
    }
  }

  return rain || n.toString();
};
